src/internal.h

```diff
@@ -432,6 +432,24 @@ DISPATCH_EXPORT DISPATCH_NOTHROW void dispatch_atfork_child(void);
 #define unlikely(x) (!!(x))
 #endif // __GNUC__
 
+#ifndef TAILQ_CONCAT
+#define	TAILQ_CONCAT(head1, head2, field) do {				\
+	if (!TAILQ_EMPTY(head2)) {					\
+		*(head1)->tqh_last = (head2)->tqh_first;		\
+		(head2)->tqh_first->field.tqe_prev = (head1)->tqh_last;	\
+		(head1)->tqh_last = (head2)->tqh_last;			\
+		TAILQ_INIT((head2));					\
+	}								\
+} while (0)
```

Comment:
This macro is entirely incorrect and wont even build.  The fields are named differently.

Reply:
This is taken straight from the macOS SDK and looks identical to me to [the implementation](https://github.com/apple/swift-corelibs-libdispatch/pull/453/files#diff-2731630646bf7c7e29667b3b6cfc03e2R95) from #453.